### PR TITLE
226-V6 to Prod

### PR DESCRIPTION
**Pull request by rnathan**

### CODE REVIEW

File: aws/prod/us-west-2-core-uswest2.yaml [hunk 1]
```diff
@@ -92,10 +105,10 @@ sitebridge_config:
...
     - sfdc.net
     - force.com
+
+# K8S Config - Start
 eks_k8s_version: '1.15'
-vpc_suffix: 1
-outbound_vpcs_config:
-  vpc_cidrs:
-    - 10.0.4.0/22
 inbound_service_name: core-uswest2-cni-inbound
```
Comment (agajjala) [inline aws/prod/us-west-2-core-uswest2.yaml:111]: These don't fall under K8S

File: aws/prod/us-east-1-core-useast1.yaml [hunk 2]
```diff
@@ -48,32 +49,46 @@ api_access_whitelist:
...
 
-outbound_vpc_cidr: 10.32.4.0/22
+# Dataplane Outbound - Start
 outbound_data_plane_node_group_instance_types:
   - m5.4xlarge
 outbound_data_plane_node_group_max_size: 6
+outbound_vpcs_config:
+  1:
+    vpc_cidr: 10.32.4.0/22
+    vpc_public_subnet_cidrs:
```
Comment (kevinwang-sfdc) [inline aws/prod/us-east-1-core-useast1.yaml:71]: This key should be removed or Terraform may think we don't wish to create any public subnets

File: aws/prod/us-east-1-core-useast1.yaml [hunk 3]
```diff
@@ -48,32 +49,46 @@ api_access_whitelist:
...
+# Dataplane Outbound - End
+
+# Payload - Start
+lambda_layer_s3_key: zip/cni_common_v226-6.zip
+lambda_function_s3_key: zip/api_v4.zip
```
Comment (kevinwang-sfdc) [inline aws/prod/us-east-1-core-useast1.yaml:76]: @pallavi-udmalpet-sfdc, should this be `zip/api_v5.zip` if we are using `v266-6`?

Comment (pallavi-udmalpet-sfdc) [reply aws/prod/us-east-1-core-useast1.yaml:76]: nope ... its correct .. should be v4.. v5 is for -7 and above..

**Comment by rnathan** [comment]
> https://github.com/sf-sdn/cni-deployer/blob/b3944f371c9f7478031bddf5a469407b675855da/Infrastructure/provider/aws/control_plane/variables.tf#L37

Good call. I have removed VPN IPs. Since we still need monitoring VM, keeping others around.

**Pull request merged**